controllers/admin/AdminBlockListingController.php

```diff
@@ -45,46 +55,51 @@ public function ajaxProcessChangeBlockStatus()
 
         $updateResult = Db::getInstance()->update('psreassurance', $dataToUpdate, $whereCondition);
 
-        $this->ajaxDie(json_encode($updateResult ? 'success' : 'error'));
+        // Response
+        $this->ajaxRenderJson($updateResult ? 'success' : 'error');
     }
 
     /**
      * Update how the blocks are displayed in the front-office
      *
      * @throws PrestaShopException
      */
-    public function ajaxProcessSavePositionByHook()
+    public function displayAjaxSavePositionByHook()
     {
         $hook = Tools::getValue('hook');
         $value = Tools::getValue('value');
-
-        if (empty($hook) || !in_array($value, array('0', '1'))) {
-            $this->ajaxDie(json_encode('error'));
+        $result = false;
+
+        if (!empty($hook) && in_array($value, array(
+                blockreassurance::POSITION_NONE,
+                blockreassurance::POSITION_BELOW_HEADER,
+                blockreassurance::POSITION_ABOVE_HEADER,
+            ))) {
```

Comment:
```suggestion
            ))
        ) {
```

Reply:
let's keep cs improvement for php-cs-fixer?

Reply:
I have no warning in PHPCSFixer (with or without). But I made the change.